src/frontend/packages/lib_components/src/common/UploadManager/index.tsx

```diff
@@ -31,6 +32,12 @@ export interface UploadingObject {
   progress: number;
   status: UploadManagerStatus;
   message?: string;
+  parentType?:
```

Comment:
Instead of having a `parentType` what do you think about creating a map with the `modelName` as key and a `parent` property containing the `parentType` ? Something like this : 

```
{
    [modelName.TIMEDTEXTTRACKS]: {
      parent: modelName.VIDEOS
    },
    [modelName.THUMBNAILS]: {
      parent: modelName.VIDEOS
    },
    ...
  }
```

And if a `parentId` is used then you search the parent in this map.

Reply:
I'm not sure to get the benefit of your proposal. We should talk about it IRL.

Reply:
I did not implement it the way you explained, but it's done here.